### PR TITLE
Module 1: Day 4 - Portfolio Project

### PR DESCRIPTION
**Pull request by Strangeal**
## Portfolio project

- Added mobile work cards and relevant styles
- Added skill list 
- Added about me

### CODE REVIEW

File: header.html [hunk 1]
```diff
@@ -25,22 +25,127 @@
...
+                        <li><a href="">javascript</a></li>
+                    </ul>
+                    <button class="card-button" type="submit">See Project</button>
+                </div>
+            </section>
+            <!--Work Cards 3-->
+            <section class="card">
+                <img src="img/Snapshoot-Portfolio2.svg" alt="Snapshoot-Portfolio">
+                <div class="card-block">
+                    <h1>Tonic</h1>
```
Comment (MaganaAsiati) [inline header.html]: - [x] Kindly replace `<h2>` with `<h1>` tag and make sure there's only one `<h1>` tag on a web page for SEO reasons. you should only have one `<h1>` in your code 👍

File: header.html [hunk 2]
```diff
@@ -25,22 +25,127 @@
...
+                    <ul class="card-tags">
+                        <li><a href="">html</a></li>
+                        <li><a href="">css</a></li>
+                        <li><a href="">javascript</a></li>
+                    </ul>
+                    <button class="card-button" type="submit">See Project</button>
+                </div>
+            </section>
+            <section class="about">
+                <h1>About Myself</h1>
```
Comment (MaganaAsiati) [inline header.html]: - [x] Kindly replace `<h2>` with `<h1>` tag and make sure there's only one `<h1>` tag on a web page for SEO reasons. you should only have one `<h1>` in your code 👍

File: header.html [hunk 3]
```diff
@@ -25,22 +25,127 @@
...
+                        <li class="no-count">CANOPY</li>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="card-tags">
+                        <li><a href="">html</a></li>
+                        <li><a href="">css</a></li>
+                        <li><a href="">javascript</a></li>
+                    </ul>
```
Comment (MaganaAsiati) [inline header.html:61]: - [x] Good job by trying to implement the languages buttons list of skills, but something went bad. you didn't implement them as on the Figma design. Kindly make it match the design.
### Your design:
WhatsApp Image 2022-08-18 at 8 42 37 PM

### Figma design:
Screen Shot 2022-08-18 at 20 54 16

File: header.html [hunk 4]
```diff
@@ -25,22 +25,127 @@
...
+                <h1>About Myself</h1>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <a href="#">LET’S CONNECT</a>
+                <ul class="social-icons">
+                    <li><a href="#"><img src="img/twitter.svg" alt="Twitter"></a></li>
+                    <li><a href="#"><img src="img/Linkedin.svg" alt="Linkedin"></a></li>
+                    <li><a href="#"><img src="img/medium.svg" alt="Medium"></a></li>
+                    <li><a href="#"><img src="img/github.svg" alt="Github"></a></li>
+                    <li><a href="#"><img src="img/Vector.svg" alt="Vector"></a></li>      
+                </ul>
```
Comment (MaganaAsiati) [inline header.html:132]: - [x] Good job by trying to implement the languages icons list of skills, but something went bad. you didn't implement them as on the Figma design, kindly fix that 👍 

## Your design

WhatsApp Image 2022-08-18 at 8 44 39 PM (1)

## Figma design

Screen Shot 2022-08-18 at 21 10 55

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 

Hi @Strangeal 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlonzoLeroneThisCloseGIF

## Highlights
✅ No linter error
✅ README is professional

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- [x] I would suggest you use good commit messages since their very descriptive while tracking the changes in your project 😁 
- [x] Kindly make your pull request more descriptive so as to let us know what you implemented in the current branch 😀 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi @Strangeal, 👋🏻👋🏻 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the functionality ✅
📌 Good commit messages ✅
📌Professional README file. ✅

 ## Required Changes ♻️
_Check the comments under the review._
- [ REQUIRED ] Kindly make all the changes that the previous code reviewer asked because it is important to flow the sugesstion to make your code more cleasn dry and match the Figma design as much as possible 🚀💯💯✨✨

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by awais-amjed — APPROVED** [review]
# STATUS: APPROVED 🥳

Hi @Strangeal , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations on successfully implementing the required changes! 🎉👏🥇✨

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] Nice Design ✔️
- [x] Good code organization ✔️ 

gif

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**